cmd/endpoint.go

```diff
@@ -142,6 +147,16 @@ func NewEndpoint(arg string) (ep Endpoint, e error) {
 	}, nil
 }
 
+// checkIPv4 - helper for validating if the provided arg is an ip address.
+func checkIPv4(ipAddress string) bool {
```

Comment:
1. Ok sure. 
2. Not clear what you mean here.
3. ah i think we can do this later when we compact the code in entire endpoint.go. If you see bala we do this check in other places in this file too. How about we wait to remove it properly everywhere and then move this somewhere else appropriate such as net.go